lib/templates/accountTemplate.js

```diff
@@ -107,21 +107,69 @@ class AccountTemplate {
     }, {});
 
     // We need to also create the mapping_list_ranges, which could be different per firm & partner
+
+    /* 
+      Mapping list ranges sample in config.json:
+      ...
+    "account_range": "280,282", // This is the "no mapping list" range
+    "mapping_list_ranges": [
+      {
+        "partner_account_mapping_list_id": 2,
+        "account_range": "280,282",
+        "type": "partner",
+        "env_id": "84"
+      },
+      {
+        "account_mapping_list_id": 8957,
+        "account_range": "280,282,284",
+        "type": "firm",
+        "env_id": "13827"
+      }
+    ],
+    ...
+    */
+
     const existingMappingRanges = existingConfig.mapping_list_ranges || [];
-    const updatedMappingRanges = [...existingMappingRanges];
+    const mappingListIds = template.mapping_list_ranges.map((range) =>
```

Comment:
A small remark: for better readability maybe rename this to `mapping_list_ids_from_server` or similar?

Reply:
Updated this to ´latestMappingListIds´

Reply:
Updated this to `latestMappingListIds` so it aligns with the renamed `latestMappingListRange` as well